### DISCUSSION TITLE
Pretraining hyperparameters for CHGNet

### DISCUSSION THREAD

**Discussion by shuix007** [CederGroupHub/chgnet#128] [Announcements]
Dear Authors,

Thank you for the great work and the effort in putting up the software! 

Is it possible that you can share the configuration (e.g., batch size, number of epochs, etc.) for training (not fine-tuning) CHGNet from scratch?

**Comment by bowen-bd** [comment]
All the hyperparameters for v0.3.0 can be found [here](https://github.com/CederGroupHub/chgnet/blob/main/chgnet/pretrained/0.3.0/README.md)

**Reply by shuix007** [reply]
Hi Bowen,

Thank you for the pointer!
